src/browser.ts

```diff
@@ -39,6 +39,8 @@ import {GrpcClientOptions, ClientStubOptions} from './grpc';
 import {GaxCall, GRPCCall} from './apitypes';
 import {Descriptor} from './descriptor';
 import {createApiCall as _createApiCall} from './createApiCall';
+
+export {PathTemplate} from './pathTemplate';
```

Comment:
we don't need it here?

Reply:
Just for reference, I added it while working on the `x-goog-request-params` stuff, it seemed required for that. But I'll remove it for now

Reply:
Or actually should I leave it in? Webpacking the DLP API, for example, only worked with this export, so would it be better to still keep it?

Reply:
Oh, I see. Yes, let's leave it then.